### DISCUSSION TITLE
Can it be used like this?

### DISCUSSION THREAD

**Discussion by xiaohucode** [fzyzcjy/flutter_rust_bridge#1686] [Q&A] [answered]
Can it be used like this?
There are many errors when using it like this

use sxd_document crate
```
pub fn parse(content: String) -> sxd_document::Package {
    sxd_document::Package::new()
}

```

**Comment by fzyzcjy** [comment, marked as answer]
Sure, it can - feel free to create a bug report if it cannot!

If you want Package to be opaque type, it should be done just like your code without modifications; if you want to access its fields directly, try https://cjycode.com/flutter_rust_bridge/guides/types/translatable/external/diff-crate.

**Reply by xiaohucode** [reply]
Thank you very much!

**Reply by fzyzcjy** [reply]
You are welcome!
